pkg/controller/quota/quota_manager_interface.go

```diff
@@ -21,7 +21,7 @@ import (
 )
 
 type QuotaManagerInterface interface {
-	Fits(aw *arbv1.AppWrapper, resources *clusterstateapi.Resource, proposedPremptions []*arbv1.AppWrapper) (bool, []*arbv1.AppWrapper, string)
+	Fits(aw *arbv1.AppWrapper, requestedResources *clusterstateapi.Resource, clusterResources *clusterstateapi.Resource, proposedPremptions []*arbv1.AppWrapper) (bool, []*arbv1.AppWrapper, string)
```

Comment:
looking at the signature of the interface, are we doing a cluster resource check in this method? isn't there a cluster resource check outside this method?